### ISSUE TITLE
research a11y scanning, in order to support itdashboard

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#630]
From [here](https://www.itdashboard.gov/faq#Federal-Web-Metrics)...

Here's the current descriptions of how the scans work:  

#### What is the “HTML Attributes” Accessibility Metric?
Accessibility Metric “HTML Attributes” reports on how many HTML tags have missing attribute text (e.g., missing/empty links and buttons, missing form input labels, missing document language) as detected by [Pa11y](https://github.com/pa11y/pa11y).

Hyper Text Markup Language (HTML) is the core markup language for the World Wide Web. HTML and other web standards describe the structure of a Web page using a series of elements (and tags and attributes) that tell browsers, search engines, translation software and assistive technology, such as screen readers how to navigate through and display Web content. HTML elements are represented by tags that give different pieces of content specific attributes, for example, “paragraph”, “table”, “heading”.

For more information about Section 508 standards please visit [www.access-board.gov/ict/](https://www.access-board.gov/ict/). For more information about the importance of HTML attributes in accessibility compliance, please visit [www.w3.org/WAI/WCAG21/Techniques/html/H91.html](https://www.w3.org/WAI/WCAG21/Techniques/html/H91.html).

#### What is the “Color Contrast” Accessibility Metric?
The “Color Contrast” accessibility scanning by [Pa11y](https://github.com/pa11y/pa11y) is used to check if a website homepage’s color contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text. The objective of this technique is to make sure that users can read text that is presented over a background.

The “Color Contrast” accessibility metric reports how many color contrast issues were found on a website homepage out of compliance with the Section 508 standards set by the U.S. Access Board, which requires agencies to conform to WCAG 2.0’s Level A and Level AA Success Criteria and Conformance Requirements.

For more information about Section 508 standards please visit [www.access-board.gov/ict/](https://www.access-board.gov/ict/). For more information about the importance of color contrast attributes in accessibility compliance, please see https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html

#### What is the “Missing Image Description” Accessibility Metric?
For blind and visually impaired individuals, all types of visual information, including Web images, are unusable unless an equivalent text description is available that screen readers can provide either by speech or braille.

The “Missing Image Description” accessibility scanning by [Pa11y](https://github.com/pa11y/pa11y) is used to check and report on how many website images and other non-text content, including user interface elements, are missing a text description.

For more information about Section 508 standards please visit [www.access-board.gov/ict/](https://www.access-board.gov/ict/). For more information about the importance of alt text in accessibility compliance, please visit https://www.w3.org/WAI/WCAG21/Techniques/html/H37.

**Comment by gbinal** [comment]
This has gone well.   Moving onto prototyping
